android/build.gradle

```
@@ -3,68 +3,26 @@
 buildscript {
 
 	ext {
-		buildToolsVersion = "30.0.3"
+		buildToolsVersion = "33.0.0"
 		minSdkVersion = project.hasProperty('minSdkVersion') ? project.getProperty('minSdkVersion') : 23
-        compileSdkVersion = 31
-         targetSdkVersion = 31
-		kotlin_version = "1.5.31"
+		compileSdkVersion = 33
+		targetSdkVersion = 33
+		// We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
+		ndkVersion = "24.0.8215888"
```

```suggestion
		ndkVersion = "23.1.7779620"
```

ref: https://github.com/MetaMask/metamask-mobile/pull/6384/files#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aL11-R11 cc @cortisiko

---

Agreed with @leotm leave this change in https://github.com/MetaMask/metamask-mobile/pull/6384 

We should pay attention to performance degradation, it might be related to using an unsupported ndk version.